app/templates/view_suppliers.html

```diff
@@ -41,6 +34,7 @@
     {% else %}
```

Comment:
Is it a good idea having the case where more permission-requiring stuff is visible being the default? The answer may well be 'yes' in this case - I don't know enough about the context really.

Reply:
Im not sure what you mean by more and less here?  Basically this page is completely different for admin-ccs-sourcing than for other admin users - sourcing get a lit of frameworks and links to framework agreement approval/download.  Other admin users get the "Change name, Users, Services" links - it's not more- or less-allowed rather than a totally different set of stuff.  They should probably be entirely different views.  I might even do that now because this is horrible.

Reply:
Yep, I think that would be easier to understand (especially for admin n00bs like me). If the templates show two completely different things, I am happy to have two different templates.